nessus-agent/windows/init.sls

```diff
@@ -6,6 +6,46 @@
 # In addiont, the sls file requires that the salt winrepo has been
 # configured with a `nessus agent` pkg definition
 
+{%- if (nessus.log_rotation_strategy) %}
+Add Log Rotation Strategy:
+  file.replace:
+    - name: 'C:\ProgramData\Tenable\Nessus Agent\nessus\log.json'
+    - pattern: '"file": "c:\\\\ProgramData\\\\Tenable\\\\Nessus Agent\\\\nessus\\\\logs\\\\www_server.log"'
+    - repl: '"rotation_strategy": {{nessus.log_rotation_strategy}},\n\t\t\t\t\g<0>'
```

Comment:
Is the json file tab-indented? That way we have `\t\t\t\t`? And is it really 4 tabs on Windows and 2 on Linux?

Reply:
I'm not sure.  I'll use spaces instead.